README.md

```diff
@@ -15,32 +15,55 @@
+| [**api.js**](https://github.com/tauri-apps/tauri/tree/dev/tooling/api)          | JS API for interaction with Rust backend | [![](https://img.shields.io/npm/v/@tauri-apps/api.svg)](https://www.npmjs.com/package/@tauri-apps/api)           | ✅  | ✅  | ✅  |
+| [**create-tauri-app**](https://github.com/tauri-apps/create-tauri-app)          | Get started with your first Tauri app    | [![](https://img.shields.io/npm/v/create-tauri-app.svg)](https://www.npmjs.com/package/create-tauri-app)         | ✅  | ✅  | ✅  |
+| [**vue-cli-plugin-tauri**](https://github.com/tauri-apps/vue-cli-plugin-tauri/) | Vue CLI plugin for Tauri                 | [![](https://img.shields.io/npm/v/vue-cli-plugin-tauri.svg)](https://www.npmjs.com/package/vue-cli-plugin-tauri) | ✅  | ✅  | ✅  |
+| [**core**](https://github.com/tauri-apps/tauri/tree/dev/core/tauri)             | runtime core                             | [![](https://img.shields.io/crates/v/tauri.svg)](https://crates.io/crates/tauri)                                 | ✅  | ✅  | ✅  |
+| [**bundler**](https://github.com/tauri-apps/tauri/tree/dev/tooling/bundler)     | manufacture the final binaries           | [![](https://img.shields.io/crates/v/tauri-bundler.svg)](https://crates.io/crates/tauri-bundler)                 | ✅  | ✅  | ✅  |
 
 ## Introduction
+
 Tauri is a framework for building tiny, blazingly fast binaries for all major desktop platforms. Developers can integrate any front-end framework that compiles to HTML, JS and CSS for building their user interface. The backend of the application is a rust-sourced binary with an API that the front-end can interact with.
 
 The user interface in Tauri apps currently leverages [`tao`](https://docs.rs/tao) as a window handling library on macOS and Windows, and [`gtk`](https://gtk-rs.org/docs/gtk/) on Linux via the **Tauri-team** incubated and maintained [WRY](https://github.com/tauri-apps/wry), which creates a unified interface to the system webview (and other goodies like Menu and Taskbar), leveraging WebKit on macOS, WebView2 on Windows and WebKitGTK on Linux.
 
 To learn more about the details of how all of these pieces fit together, please consult this [ARCHITECTURE.md](https://github.com/tauri-apps/tauri/blob/dev/ARCHITECTURE.md) document.
 
 ## Get Started
+
 If you are interested in making a tauri app, please visit the [documentation website](https://tauri.studio). This README is directed towards those who are interested in contributing to the core library. But if you just want a quick overview about where `tauri` is at in its development, here's a quick burndown:
 
 ### Platforms
-- [x] Windows 7,8,10
-- [x] Linux
-- [x] macOS
-- [ ] iOS (in progress)
-- [ ] android (soon)
+
+Tauri currently supports development and distribution on the following platforms:
+
+| Platform                 | Versions                |
+| :----------------------- | :---------------------- |
+| Windows                  | 7 (64-bit), 8 and above |
```

Comment:
_Building_ works only on 64bit no matter the windows version.
_Running_ should work on 32bit no matter the windows version, can't test it right now tho.